projects/ad400x-st/README.rst

```diff
@@ -0,0 +1,104 @@
+Evaluating AD400x on SDP-K1
```

Comment:
documentation usually goes in separate commit.

Reply:
ok.